### ISSUE TITLE
Containers are up, but Kathara can't find any labs

### ISSUE THREAD

**Issue by zhangineer** [KatharaFramework/Kathara-Labs#11]
I am unable to retrieve info of a started lab, nor can I connect to any devices using the kathara commands
```
kathara linfo
CRITICAL - Lab is not started.

docker ps
CONTAINER ID   IMAGE                               COMMAND                  CREATED       STATUS        PORTS                                                                                                                    NAMES
4642fecba145   kathara/frr                         "bash"                   2 days ago    Up 2 days                                                                                                                              kathara_ubuntu-brtxaxo3btr5dbjzdzeuyq_r5_WLfspmcnPKd7iQMaFVoVdw
3e0c4591df23   kathara/frr                         "bash"                   2 days ago    Up 2 days                                                                                                                              kathara_ubuntu-brtxaxo3btr5dbjzdzeuyq_r3_WLfspmcnPKd7iQMaFVoVdw
6e679c0ecaa3   kathara/frr                         "bash"                   2 days ago    Up 2 days                                                                                                                              kathara_ubuntu-brtxaxo3btr5dbjzdzeuyq_r1_WLfspmcnPKd7iQMaFVoVdw
4375873469a7   kathara/frr                         "bash"                   2 days ago    Up 2 days                                                                                                                              kathara_ubuntu-brtxaxo3btr5dbjzdzeuyq_r4_WLfspmcnPKd7iQMaFVoVdw
e5e4515f7c8a   kathara/frr                         "bash"                   2 days ago    Up 2 days                                                                                                                              kathara_ubuntu-brtxaxo3btr5dbjzdzeuyq_r2_WLfspmcnPKd7iQMaFVoVdw
```

I was able to get into the containers by using bash commands though. The issues seems to be only related to the kathara command. 

I don't have this issue immediately after starting kathara. So it only occured after some time after the containers started

**Comment by Skazza94** [comment]
Hi @zhangineer,
could you please share further details about the problem?
You can run `kathara linfo` only when you are inside a lab folder (that means a folder with a `lab.conf` file).

**Comment by Skazza94** [comment]
Hi @zhangineer,
this problem could happen when the lab is started and then the directory is renamed (since the `lab_hash` is computed from the lab path). Did you renamed/moved the folder?

I suggest to use the `LAB_NAME` meta in the `lab.conf` file to avoid such problems. 

The next Kathara release (currently in `develop` branch) includes an improvement (see issue [#163](https://github.com/KatharaFramework/Kathara/issues/163)): the `lab_hash` is computed from the `LAB_NAME` meta if present, otherwise fallbacks to the lab path.

**Comment by tcaiazzi** [comment]
Hi @zhangineer,

Any news about this?

**Comment by tcaiazzi** [comment]
Yes we included this feature in the current release ([3.5.1](https://github.com/KatharaFramework/Kathara/releases/tag/3.5.1))!

I close the issue! If you have some problems, feel free to reopen it. :smile: 

Many thanks!
